docs/update.md

```diff
@@ -0,0 +1,295 @@
+
+2. Update the [Custom Resource Definition](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/)
+    for the Operator, taking it from the official repository on Github, and do
+    the same for the Role-based access control:
+
+    ``` {.bash data-prompt="$" }
+    $ kubectl apply -f https://raw.githubusercontent.com/percona/percona-server-mysql-operator/v{{ release }}/deploy/crd.yaml
+    $ kubectl apply -f https://raw.githubusercontent.com/percona/percona-server-mysql-operator/v{{ release }}/deploy/rbac.yaml
+    ```
+
+!!! note
+
+    You can use `helm upgrade` to upgrade the Operator only. The Database Management System (Percona Server for MySQL) should be upgraded in the same way whether you used helm to install it or not.
+
+## Upgrading Percona Server for MySQL
+
+The following section presumes that you are upgrading your cluster within the
+*Smart Update strategy*, when the Operator controls how the objects
+are updated. Smart Update strategy is on when the `updateStrategy` key in the
+[Custom Resource](operator.md) configuration file is set to `SmartUpdate`
+(this is the default value and the recommended way for upgrades).
+
+!!! note
+
+    As an alternative, the `updateStrategy` key can be set to `RollingUpdate` 
+    and `OnDelete`. You can find out more about it in the
+    [appropriate section](update.md#more-on-upgrade-strategies).
+
+### Manual upgrade
+
+Manual update of Percona Server for MySQL can be done as follows:
+
+1. Make sure that `spec.updateStrategy` option in the [Custom Resource](operator.md)
+    is set to `SmartUpdate`, `spec.upgradeOptions.apply` option is set to `Never`
+    or `Disabled` (this means that the Operator will not carry on upgrades
+    automatically).
+    
+    ```yaml
+    ...
+    spec:
+      updateStrategy: SmartUpdate
+      upgradeOptions:
+        apply: Disabled
+        ...
+    ```
+
+2. Now [apply a patch](https://kubernetes.io/docs/tasks/run-application/update-api-object-kubectl-patch/)
+    to your Custom Resource, setting necessary Custom Resource version and image
+    names with a newer version tag.
+
+    !!! note
+
+        Check the version of the Operator you have in your Kubernetes
+        environment. Please refer to the [Operator upgrade guide](update.md#upgrading-the-operator)
+        to upgrade the Operator and CRD, if needed.
+
+    Patching Custom Resource is done with the `kubectl patch ps` command.
+    Actual image names can be found [in the list of certified images](images.md)
+    (for older releases, please refer to the [old releases documentation archive](https://docs.percona.com/legacy-documentation)).
+    For example, updating `cluster1` cluster to the `{{ release }}` version
+    should look as follows:
+
+    ```bash
+    $ kubectl patch ps cluster1 --type=merge --patch '{
+       "spec": {
+           "crVersion":"{{ release }}",
+           "mysql":{ "image": "percona/percona-server-mysql-operator:{{ release }}" },
```

Comment:
probably we need more images here

Reply:
First, for `mysql.image` we need `percona/percona-server:8.0.32-24`.

The rest of the images:
proxy.haproxy.image: percona/haproxy:2.5.12
proxy.router.image: percona/percona-mysql-router:8.0.32
orchestrator.image: percona/percona-orchestrator:3.2.6-8
backup.image: percona/percona-xtrabackup:8.0.32-25
toolkit.image: percona/percona-server-mysql-operator:0.6.0-toolkit 

@hors @tplavcic could you check this list as well pls.